composer.json

```diff
@@ -4,24 +4,28 @@
     "type": "project",
     "license": "proprietary",
     "require": {
-        "php": "^8.0",
+        "php": "^8.2",
         "dvsa/mot-cpms-notifications": "^1.0.0",
         "laminas/laminas-mvc": "^3.3.0",
-        "laminas/laminas-cache": "^3.1.3",
+        "laminas/laminas-cache": "^3.10.1",
         "laminas/laminas-log": "^2.13",
         "laminas/laminas-filter": "^2.13",
-        "laminas/laminas-dependency-plugin": "^2.2",
-        "laminas/laminas-cache-storage-adapter-filesystem": "^2.0",
-        "laminas/laminas-cache-storage-adapter-memory": "^2.0",
-        "laminas/laminas-cache-storage-adapter-apcu": "^2.0"
+        "laminas/laminas-dependency-plugin": "^2.6.0",
+        "laminas/laminas-cache-storage-adapter-filesystem": "^2.3.0",
+        "laminas/laminas-cache-storage-adapter-memory": "^2.2.0",
+        "laminas/laminas-cache-storage-adapter-apcu": "^2.4.0",
+        "laminas/laminas-component-installer": "^3.4"
```

Comment:
any reason why was it moved from require-dev?

Reply:
change back